blog/announcing-new-branding/README.md

```diff
@@ -0,0 +1,97 @@
+
+#### Body
+
+The Problem
+-----------
+
+Before the process of developing our new brand identity had begun, we were frustrated by the fact that the Joystream visual identity was lacking in many respects. It was incomplete and undefined, which led to problems in communication with our audience across all of our channels, products and messages.
+
+![](branding-collage.png)
+
+The problems we faced in this area included but were not limited to:
+
+-   The logo was inconsistent (sometimes it was our old elephant logo, sometimes a wordmark, sometimes an emblem and occasionally combinations of all three)
+-   Logo usage guidelines were unclear in terms of positioning, backgrounds, etc.
+-   Key brand colors and typefaces were undefined
+-   There were no rules for photo style or illustration style in our communications (e.g. blog posts)
+-   Icons and other assets lacked a consistent style
+-   Assets for certain purposes simply did not exist
+-   We had no established brand design guidelines or a style guide
+
+You can see [more of our discussion](https://github.com/joystream/design/issues?q=is%3Aissue+is%3Aclosed) about these issues in our design repo on GitHub.
+
+The Solution
+------------
+
+We could see that an urgent solution was necessary to tackle the issues we faced with our branding. As a result, we got in contact with a team of talented designers who were able to completely overhaul our visual brand identity. This was a complicated process which lasted for an extended period of time, but we are very pleased with the results which are now ready to be deployed across our profiles and web properties.
+
+Overall, our rebranding project involved:
+
+-   Designing a new Joystream logo
+-   Determining our brand colors and fonts
+-   Redesigning our main website (joystream.org)
+-   Creating new assets for our social media profiles
+-   Creating new assets for our GitHub repositories
+-   Redesigning our blog and the assets used for blog posts
+-   Designing assets to be used in advertising
+-   Creating a "style guide" for Joystream
+
+Most of these changes will be pushed over the next few days. We hope that our community and stakeholders will be pleased with what they see!
+
+The Results
+-----------
+
+Instead of using this post to show you *all* of the new assets, logos and icons which you can expect to see deployed across our public-facing profiles and products, we thought it might be more helpful to share with you our new `Branding Guide`. This document offers a visual overview of the changes to the Joystream brand and provides some useful context and information about our new asset types.
+
+```
+
+
+
+
+EMBEDDED PDF HERE
+
+
+
+
+```
+
+If you're using an older web browser, the PDF might not display correctly. If this is the case, please click [here](http://151.236.222.69/assets/brandguide.pdf) to download the guide and view it locally on your device.
```

Comment:
Maybe add a comment here so you remember to change URL.

Reply:
Good suggestion :)